doc/reports/spikes/metrics-what-we-want-spike.md

```diff
@@ -0,0 +1,18 @@
+# Monitoring Performance - What should we monitor?
+## Introduction
+This spike report follows on from work and discussions in the OAT meeting to identify areas where we want to be monitoring the system. This outlines what I see as an idealised monitoring setup - there may be blockers to overcome to achieve some of this. I've also provided guidance in the document to how I see the monitoring being provided and used.
+## Production metrics
+All of these metrics should be radiated on monitors
+### Performance
+* **99% call time** - This can be pulled from Google Analytics, CloudWatch or Zipkin headers in Splunk logs, potentially further broken down by microservice
+* **Current concurrent users** - This can be pulled most easily from Google Analytics
```

Comment:
Need to define what "concurrent users" means

Reply:
Presumably concurrent users would be defined by whichever platform we're pulling this metric from, so it's hard for me to provide a specific definition here.